spec/koans.js

```diff
@@ -270,43 +288,45 @@ describe("the JavaScript language", function() {
       var z = true;
 
       example(x);
-      //expect(x).toEqual();
+      expect(x).toEqual(1);
 
       example(y);
-      //expect(y).toEqual();
+      expect(y).toEqual("example");
 
       example(z);
-      //expect(z).toEqual();
+      expect(z).toEqual(true);
     });
 
     it("passes arrays by reference", function() {
       function example(arg) {
         arg[0] = 100;
       }
 
-      var x = [1,2,3];
+      var x = [1, 2, 3];
 
       example(x);
-      //expect(x).toEqual();
+      expect(x).toEqual([100, 2, 3]);
     });
 
     it("passes objects by reference", function() {
```

Comment:
Almost this :) 
You've written `expect(x).toEqual(arg.property);` instead of `expect(x).toEqual({ property: 'test' });`
The correction is : 
```
    it('passes objects by reference', function () {
      function example(arg) {
        arg.property = 'test';
      }

      var x = { property: 'cool!' };

      example(x);
      expect(x).toEqual({ property: 'test' });
    });
```